colossalai/pipeline/schedule/v_schedule.py

```diff
@@ -0,0 +1,468 @@
+# Refer from Zero Bubble Pipeline Parallelism.
+# Github: https://github.com/sail-sg/zero-bubble-pipeline-parallelism
```

Comment:
add their license to ours

Reply:
> add their license to ours

Added in https://github.com/hpcaitech/ColossalAI/pull/6034/commits/8eb6eac2253a31d80a72ca4bb8e0266c75af5d10;